test/test_scanner.py

```diff
@@ -181,19 +191,6 @@ def test_openssl_1_0_2g(self):
             "CVE-2016-0800",
             ])
 
-    def test_png_1_4_11(self):
-        """Scanning test-png-1_4_11.out"""
-        self._binary_test('test-png-1.4.11.out', 'png', '1.4.11', [
-            # Check for known cves in this version
-            "CVE-2016-10087",
```

Comment:
@bksahu Why are you deleting this ?

Reply:
It's not for that version.

Reply:
"The png_set_text_2 function in libpng 0.71 before 1.0.67, 1.2.x before 1.2.57, **1.4.x before 1.4.20**, 1.5.x before 1.5.28, and 1.6.x before 1.6.27 allows context-dependent attackers to cause a NULL pointer dereference vectors involving loading a text chunk into a png structure, removing the text, and then adding another text chunk to the structure."

If there is an error with the other test cases please make a separate pull request.

Reply:
Okay sorry. I deleted it while solving merge conflict

Reply:
Let me create a separate PR.